azure-devops/ecommerce-projects/00_secrets_prod.tf

```diff
@@ -0,0 +1,21 @@
+#
+# PROD KEYVAULT
+#
+
+module "secrets" {
+  source = "git::https://github.com/pagopa/azurerm.git//key_vault_secrets_query?ref=v2.0.4"
+
+  resource_group = local.prod_key_vault_resource_group
+  key_vault_name = local.prod_key_vault_azdo_name
+
+  secrets = [
+    "DANGER-GITHUB-API-TOKEN",
+    "azure-devops-github-ro-TOKEN",
+    "azure-devops-github-rw-TOKEN",
+    "azure-devops-github-pr-TOKEN",
+    "azure-devops-github-EMAIL",
+    "azure-devops-github-USERNAME",
+    "TENANTID",
+    "DEV-SUBSCRIPTION-ID"
```

Comment:
Is it correct that `DEV-SUBSCRIPTION-ID` is here?

Reply:
yes, it is used to create a service connection for Azure Container Registry in DEV